### ISSUE TITLE
License choice discussion

### ISSUE THREAD

**Issue by Arushi282** [UBC-MDS/clevercloudr#34]

**Comment by Vikiano** [comment]
I am suggesting the use of MIT license for simplicity

**Comment by adrianne-l** [comment]
I agree with Victor. It also allows users to distribute the closed source package for their use.

**Comment by ajuliettet** [comment]
We have three common choices for licenses according to this website https://choosealicense.com/ suggested by Github. I think MIT is the most permissible. Since our project is for educational purpose, we are not doing this for any community or for improvements and updates in the future, it is a good idea to keep it simple and permissible. I agree with @Vikiano and @adrianne-l  :)

**Comment by ajuliettet** [comment]
I just noticed that MIT license actually allows commercial usage. Do you feel comfortable with that? Let's discuss!

**Comment by Arushi282** [comment]
Hi @aimee0317, that is actually a very interesting observation and thank you for bringing that to light. I personally don't have any issue with the software being sold, so along as they comply with the rest of the license and copyright conditions. Commercial use can be seen in many different forms, and so it would be interesting to see how our package can be used for financial gain.

**Comment by adrianne-l** [comment]
@aimee0317 @Arushi282 It would be flattering to be used in commercial context and I support Arushi's view on this.

**Comment by ajuliettet** [comment]
Great discussion here on commercial usage! I think the only other license that might make sense is this copyleft license https://choosealicense.com/licenses/gpl-3.0/, but it's more restrictive according to this article https://www.whitesourcesoftware.com/resources/blog/top-10-gpl-license-questions-answered/. If you do not mind commercial usage, I think we will use MIT!

**Comment by Vikiano** [comment]
@aimee0317, this is spot on. In terms of restriction, I also think MIT is the least restrictive. So, MIT, yes!
